devops/tests/test_tasks.py

```diff
@@ -107,6 +108,7 @@
     Tabw4C6DPpfMA3XlhJkhAkEAoIcAcIwMxj2i46WdlSL8zt/5EAgeF0jdCLJPU6J5
     xrbc46CIEyiNKpyhIdDOcZqsUevytVTyOxSnnsOYBdW5LA==
     -----END RSA PRIVATE KEY-----
+  some_missing_value:
```

Comment:
Should there also be one with an actual `null` value to test?

Reply:
Null can be expressed in multiple ways in YAML, that is actually a null value.

Reply:
I could add tat as this is an unsealed version of the file and we're using ruamel.yaml to output the unsealed version and it dumps the null values this way, I'd prefer to keep it this way rather than using mixed representations.

Reply:
OK.